stubs/braintree/braintree/customer.pyi

```diff
@@ -39,17 +38,30 @@ class Customer(Resource):
     def create_signature(): ...
     @staticmethod
     def update_signature(): ...
-    payment_methods: Any
-    credit_cards: Any
-    addresses: Any
-    paypal_accounts: Any
-    apple_pay_cards: Any
-    android_pay_cards: Any
-    amex_express_checkout_cards: Any
-    europe_bank_accounts: Any
-    venmo_accounts: Any
-    us_bank_accounts: Any
-    visa_checkout_cards: Any
-    masterpass_cards: Any
-    samsung_pay_cards: Any
+    payment_methods: list[
+        CreditCard
+        | Address
+        | PayPalAccount
+        | ApplePayCard
+        | AndroidPayCard
+        | AmexExpressCheckoutCard
+        | EuropeBankAccount
+        | VenmoAccount
+        | UsBankAccount
+        | VisaCheckoutCard
+        | MasterpassCard
+        | SamsungPayCard
+    ]
```

Comment:
```suggestion
    payment_methods: list[Resource]
```

Since you'd have to use `isinstance` anyway for most usage, and indidual types are already available in their own separate lists, it makes sense to simplify this definition